### PR TITLE
Upgrade .NET SDK to 6.0.100

### PR DESCRIPTION
**Pull request by patriksvensson**
Also upgrades Cake to version 2.0.0-rc0001

### CODE REVIEW

File: build.cake [hunk 1]
```diff
@@ -38,7 +38,7 @@ Task("Build-Examples")
     .IsDependentOn("Build")
     .Does(context => 
 {
-    DotNetCoreBuild("./examples/Examples.sln", new DotNetCoreBuildSettings {
+    DotNetBuild("./examples/Examples.sln", new DotNetCoreBuildSettings {
```
Comment (nils-a) [inline build.cake:41]: Nitpick: 
```suggestion
    DotNetBuild("./examples/Examples.sln", new DotNetBuildSettings {
```

File: build.cake [hunk 2]
```diff
@@ -14,7 +14,7 @@ Task("Build")
     .IsDependentOn("Clean")
     .Does(context => 
 {
-    DotNetCoreBuild("./src/Spectre.Console.sln", new DotNetCoreBuildSettings {
+    DotNetBuild("./src/Spectre.Console.sln", new DotNetCoreBuildSettings {
```
Comment (nils-a) [inline build.cake:17]: Nitpick:
```suggestion
    DotNetBuild("./src/Spectre.Console.sln", new DotNetBuildSettings {
```

File: build.cake [hunk 3]
```diff
@@ -26,7 +26,7 @@ Task("Build-Analyzer")
     .IsDependentOn("Build")
     .Does(context => 
 {
-    DotNetCoreBuild("./src/Spectre.Console.Analyzer.sln", new DotNetCoreBuildSettings {
+    DotNetBuild("./src/Spectre.Console.Analyzer.sln", new DotNetCoreBuildSettings {
```
Comment (nils-a) [inline build.cake:29]: Nitpick:
```suggestion
    DotNetBuild("./src/Spectre.Console.Analyzer.sln", new DotNetBuildSettings {
```

File: build.cake [hunk 4]
```diff
@@ -52,13 +52,13 @@ Task("Test")
     .IsDependentOn("Build-Examples")
     .Does(context => 
 {
-    DotNetCoreTest("./test/Spectre.Console.Tests/Spectre.Console.Tests.csproj", new DotNetCoreTestSettings {
+    DotNetTest("./test/Spectre.Console.Tests/Spectre.Console.Tests.csproj", new DotNetCoreTestSettings {
```
Comment (nils-a) [inline build.cake:55]: Nitpick:
```suggestion
    DotNetTest("./test/Spectre.Console.Tests/Spectre.Console.Tests.csproj", new DotNetTestSettings {
```

File: build.cake [hunk 5]
```diff
@@ -52,13 +52,13 @@ Task("Test")
...
         Configuration = configuration,
         NoRestore = true,
         NoBuild = true,
     });
 
-    DotNetCoreTest("./test/Spectre.Console.Analyzer.Tests/Spectre.Console.Analyzer.Tests.csproj", new DotNetCoreTestSettings {
+    DotNetTest("./test/Spectre.Console.Analyzer.Tests/Spectre.Console.Analyzer.Tests.csproj", new DotNetCoreTestSettings {
```
Comment (nils-a) [inline build.cake:61]:
```suggestion
    DotNetTest("./test/Spectre.Console.Analyzer.Tests/Spectre.Console.Analyzer.Tests.csproj", new DotNetTestSettings {
```

File: build.cake [hunk 6]
```diff
@@ -69,7 +69,7 @@ Task("Package")
     .IsDependentOn("Test")
     .Does(context => 
 {
-    context.DotNetCorePack($"./src/Spectre.Console.sln", new DotNetCorePackSettings {
+    context.DotNetPack($"./src/Spectre.Console.sln", new DotNetCorePackSettings {
```
Comment (nils-a) [inline build.cake:72]:
```suggestion
    context.DotNetPack($"./src/Spectre.Console.sln", new DotNetPackSettings {
```

File: build.cake [hunk 7]
```diff
@@ -78,7 +78,7 @@ Task("Package")
             .TreatAllWarningsAs(MSBuildTreatAllWarningsAs.Error)
     });
 
-    context.DotNetCorePack($"./src/Spectre.Console.Analyzer.sln", new DotNetCorePackSettings {
+    context.DotNetPack($"./src/Spectre.Console.Analyzer.sln", new DotNetCorePackSettings {
```
Comment (nils-a) [inline build.cake:81]:
```suggestion
    context.DotNetPack($"./src/Spectre.Console.Analyzer.sln", new DotNetPackSettings {
```

File: build.cake [hunk 8]
```diff
@@ -134,7 +134,7 @@ Task("Publish-NuGet")
     foreach(var file in context.GetFiles("./.artifacts/*.nupkg")) 
     {
         context.Information("Publishing {0}...", file.GetFilename().FullPath);
-        DotNetCoreNuGetPush(file.FullPath, new DotNetCoreNuGetPushSettings
+        DotNetNuGetPush(file.FullPath, new DotNetCoreNuGetPushSettings
```
Comment (nils-a) [inline build.cake:137]:
```suggestion
        DotNetNuGetPush(file.FullPath, new DotNetNuGetPushSettings
```

**Review by nils-a — APPROVED** [review]
Couple of nitpicks. Everything else LGTM!

**Comment by patriksvensson** [comment]
@nils-a Done!

**Pull request merged**